src/collection.jl

```diff
@@ -9,6 +9,16 @@ end
 
 parse_name(filename, ext, ::Val{true}) = filename
 
+
+# transposes the data and saves it, the reason is same as that for getdata
+function setdata(file_path, data)
+    d = ndims(data)
+    transposed_data = permutedims(data, d:-1:1)
+    FITS(file_path, "w") do fh
+        write(fh, transposed_data)
+    end
+end
+
 #---------------------------------------------------------------------------------------
 @doc raw"""
     fitscollection(dir; recursive=true, abspath=true, keepext=true, ext=r"fits(\.tar\.gz)?", exclude=nothing, exclude_dir=nothing, exclude_key = ("", "HISTORY"))
```

Comment:
I'm not sure how this actually looks in the docs, but I imagine it renders poorly with a horizontal scroll bar. If that is the case, I would strongly recommend breaking up the kwargs with newlines